app.rb

```diff
@@ -4,17 +4,18 @@
 require_relative 'book'
 require_relative 'rental'
 require_relative 'classroom'
-require_relative 'options'
-
-
+require './write_read'
 class App
   def initialize
     @books = []
     @people = []
     @rentals = []
+
```

Comment:
- Kindly fix the error when a user tries to run the application and there's no JSON files already created

For example, if in the program there's nothing stored or already created, the program automatically should create the necessary file when you create a user or books or rental, but shouldn't give you an error while trying to run the application without JSON files created

Reply:
@BregornOriginal ,  the partner has posted the video of the same, deleting all the files and starting from the scratch, Kindly elaborate further..